types/types.go

```diff
@@ -88,7 +88,7 @@ type ServiceConfig struct {
 	Name     string   `yaml:"-" json:"-"`
 	Profiles []string `mapstructure:"profiles" yaml:"profiles,omitempty" json:"profiles,omitempty"`
 
-	Build        *BuildConfig `yaml:",omitempty" json:"build,omitempty"`
+	Build        *BuildConfig `yaml:"build,omitempty" json:"build,omitempty"`
```

Comment:
doesn't hurt but not required, is this intentional ?

Reply:
ah no, I put it there while trying to figure out what was missing and forgot it 😅 I'll remove it